src/view.tsx

```diff
@@ -0,0 +1,50 @@
+import { VDomRenderer } from '@jupyterlab/apputils';
+/*import { ReactWidget } from '@jupyterlab/apputils';*/
+import { ThemeEditorModel } from './model';
+import React from 'react';
+import Form from '@rjsf/core';
+
+interface IProps {
+  formData: any;
+  schema: any;
+  uiSchema: any;
+  setformData: (value: any) => void;
+}
+
+function FormComponent(props: IProps) {
+  return (
+    <Form
+      schema={props.schema}
+      formData={props.formData}
+      uiSchema={props.uiSchema}
+      onChange={event => {
+        props.setformData(event.formData);
+      }}
+    />
+  );
+}
+
+export class ThemeEditorView extends VDomRenderer<ThemeEditorModel> {
+  public formData: any;
+  public schema: any;
+  public uiSchema: any;
+  constructor(model: ThemeEditorModel) {
+    super(model);
+    this.formData = this.model.formData;
+    this.schema = this.model.schema;
+    this.uiSchema = this.model.uiSchema;
+  }
+  render() {
+    return (
+      <FormComponent
+        schema={this.schema}
+        formData={this.formData}
+        uiSchema={this.uiSchema}
+        setformData={(value: any) => {
+          this.formData = value;
```

Comment:
The code is almost correct what is missing is setting the new `formData` in the model. For now what is happening is that you get a initial version of `formData` from the model. But then when it changes, the new value is stored in the view (via `this.formData`) that don't match any longer `this.model.formData`.

Reply:
Ok thanks for the comments. I suspected I had to set the formData in the model